chapters/annotations.tex

```diff
@@ -1420,14 +1420,14 @@ \subsection{Version Numbering}\label{version-numbering}
 Version numbers are of the forms:
 \begin{itemize}
 \item
-  Main release versions: """ \lstinline!UNSIGNED-INTEGER! \{ "." \lstinline!UNSIGNED-INTEGER! \} """\\
-  Example: "2.1"
+  Main release versions: \lstinline!""" UNSIGNED-INTEGER { "." UNSIGNED-INTEGER } """!\\
+  Example: \lstinline!"2.1"!
 \item
-  Pre-release versions: """ \lstinline!UNSIGNED-INTEGER! \{ "." \lstinline!UNSIGNED-INTEGER! \} " " \{\lstinline!S-CHAR!\} """\\
-  Example: "2.1 Beta 1"
+  Pre-release versions: \lstinline!""" UNSIGNED-INTEGER { "." UNSIGNED-INTEGER } " " {S-CHAR} """!\\
+  Example: \lstinline!"2.1 Beta 1"!
 \item
-  Un-ordered versions: """ \lstinline!NON-DIGIT! \{\lstinline!S-CHAR!\} """\\
-  Example: "Test 1"
+  Un-ordered versions: \lstinline!""" NON-DIGIT {S-CHAR} """!\\
```

Comment:
Similarly:
```suggestion
  Un-ordered versions: \lstinline[language=grammar]!""" NON-DIGIT {S-CHAR} """!\\
```

Reply:
Sorry, didn't see the comment in the commit message until now. But why not mark it as _grammar_ when we know it is?